docs/APIContract.md

```diff
@@ -0,0 +1,67 @@
+# Versions
+
+So far there is only one version
+
+The base URL for an API should be: _baseURL_/_apiVersion_/ meaning even something such as sometestbuildsforpoe.gg/something/a/b/c/ would be valid as a base URL, as long  as it has afterwards the per version required endpoints.
+
+## Metadata Endpoint
+
+In order to provide a consistent API a metadata endpoint should be provided under _baseURL_/.well-known/pathofbuilding this endpoint should provide information about the API and the version of the API.
+Furthermore this may allow having custom endpoints for the API.
+
+| Feature          | Field         | Type | Description                                                                       |
+| ---------------- | ------------- | ---- | --------------------------------------------------------------------------------- |
+| League Filtering | league_filter | bool | This can  be used to indicate whether the API supports filtering based on leagues |
+| Gem Filtering    | gem_filter    | bool | This can  be used to indicate whether the API supports filtering based on gems    |
+
+## Version 1
+
+### Response and Request types
+
+#### BuildInfo
+
+| Field        | Type    | Description                         |
+| ------------ | ------- | ----------------------------------- |
+| pobdata      | string  | The Path of Building data           |
+| name         | string  | The name of the build               |
+| lastModified | integer | The last modification timestamp     |
+| buildId      | string  | The unique identifier for the build |
```

Comment:
Unique identifier on the website? Is that a link or just an internal id?

Reply:
This should be an unique identifier as it could be used for updating a local cache for said build, it doesn't have to be an internal id to either pob nor the api but it needs to map to exactly one build.

As going further there may be an option long-term to sync builds from sources to PoB and updating with the click of a button instead of manually reimporting.

tl;dr: Unique Identifier provided by source to map to one specific build